test/unit/mock/profiles/dependencies/profile_c/libraries/gordon_config.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+puts "Loading gordon_config from c"
+
+# Custom resource based on the InSpec resource DSL
+class GordonConfig < Inspec.resource(1)
+  name 'gordon_config'
+
+  desc "
+    Gordon's resource description ...
+  "
+
+  example "
+    describe gordon_config do
+      its('version') { should eq('1.0') }
+      its('file_size') { should > 1 }
+    end
+  "
+
+  # Load the configuration file on initialization
+  def initialize
+    @params = {}
+    @path = '/tmp/gordon/config.yaml'
```

Comment:
we may simplify the resource to just return two different version numbers. Otherwise we are relying on external configuration.